airflow/utils/email.py

```diff
@@ -87,7 +106,7 @@ def send_email_smtp(
     from_email: str | None = None,
     custom_headers: dict[str, Any] | None = None,
     **kwargs,
-):
+) -> None:
```

Comment:
Maybe a good idea to add parameter docstring here for consistency? Seems to be the last undocumented method here?

Reply:
Yes, you are right, @potiuk!